frontend/components/ClickReserveButton/style.js

```diff
@@ -0,0 +1,117 @@
+import { css } from 'glamor';
+import {
+  themeConfig,
+  themeName,
+} from '@shopgate/pwa-common/helpers/config';
+
+const isIOS = themeName.includes('ios');
+const borderRadius = isIOS ? '5px' : '0px';
+
+// Override custom colors
+css.global('#rr-omni #rr-omni-custom, #rr-omni-reserve-button, #rr-inventory-custom, #rr-inventory', {
+  '--rr-color-primary': themeConfig.colors.cta,
+  '--rr-color-primary-contrast': themeConfig.colors.ctaContrast,
+  '--rr-color-link': themeConfig.colors.cta,
+  '--rr-color-button-background-disabled': themeConfig.colors.shade5,
+  '--rr-color-state-alarm': themeConfig.colors.error,
+  '--rr-color-state-warning': themeConfig.colors.ff9300,
+  '--rr-color-state-success': themeConfig.colors.success,
+  '--rr-font-family': themeConfig.typography.family,
+});
+
+// Custom Styling
+css.global('div.rr-modal-base', {
+  paddingTop: isIOS ? 'calc(1.25rem + var(--safe-area-inset-top)) !important' : '',
```

Comment:
The safe-area-insets come from the device - not the theme. This implementation causes some issues.
- when the extension is deployed for a shop that uses the GMD theme on iOS devices with notch, insets are not set and the close button will be unreachable again
- when the iOS theme is used on an iOS device without notch, there will be an extra padding of 1.25rem at the top

As far as i see, the modal also looks good without the 1.25rem on notch devices, so i'd recommend to replace this line with `paddingTop: 'var(--safe-area-inset-top) !important',`

Reply:
fixed